sagemaker-python-sdk/tensorboard_keras/tensorboard_keras.ipynb

```diff
@@ -134,45 +134,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Typo - <code>teh</code>  -&gt; <code>the</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2956/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
fixed